Makefile

```diff
@@ -154,7 +154,7 @@ pull/%: ## pull a jupyter image
 
 push/%: DARGS?=
 push/%: ## push all tags for a jupyter image
```

Comment:
Now it perfectly matches the description :)

Reply:
😄